packages/components/src/components/numpad/numpad.js

```diff
@@ -0,0 +1,36 @@
+import { useState, useEffect } from 'react';
+
+export const useLongPress = (callback = () => {}, ms = 300) => {
+    const [startLongPress, setStartLongPress] = useState(false);
+    const preventDefaults = e => {
+        e.preventDefault();
+        e.stopPropagation();
+    };
+
+    useEffect(() => {
+        let timer;
+        if (startLongPress) {
+            timer = setTimeout(callback, ms);
+        } else {
+            clearTimeout(timer);
+        }
+
+        return () => {
+            clearTimeout(timer);
+        };
+    }, [startLongPress]);
+
+    return {
+        onMouseDown: e => {
+            preventDefaults(e);
```

Comment:
should this be `preventDefault` instead of `preventDefaults` ?

Reply:
@kelly-binary , `preventDefaults` is a combination of `preventDefault` and `stopPropagation` to stop bubbling of events on `touchHold` and `mouseDown`, refer to here: [L3-L8](https://github.com/binary-com/deriv-app/pull/882/files#diff-d63521f3bde4c2dc483ae42d39ed9be5R3-R8)

Reply:
ahh i see